### ISSUE TITLE
Translations get trimmed in development

### ISSUE THREAD

**Issue by Kernle32DLL** [lingui/js-lingui#205]
Something unexpected I noticed today: Translations get trimmed, when using `core.development.js`.

The culprit is in the processTokens method [here](https://github.com/lingui/js-lingui/blob/0d0aa92e334909a27ec0a38b2c99559e3c4558e5/packages/core/src/dev/compile.js#L8). However, the cli api does not seem to trim (see [here](https://github.com/lingui/js-lingui/blob/f593b06f8cb59089adcc6b7922eecbdd69db5ba4/packages/cli/src/api/compile.js#L16)).

Not trimming is definitely the right thing to do, as otherwise intermediate translations like " and " (notice the spaces) get trimmed to "and".

**Comment by tricoder42** [comment]
Ha, good catch! You're right, this should be consistent and trimming in development should be removed.

However, trimming will remain in JSX (see [here](https://github.com/lingui/js-lingui/blob/master/packages/babel-plugin-transform-react/src/index.js#L390)), so it's consistent with how JSX works in general:

```jsx
<h1>   Helllo World   </h1>
// rendered as <h1>Hello World</h1>
```

```jsx
<Trans>    Hello World   </Trans>
// becomes <Trans id="Hello World" />
```

The core library, however, should work as you described.

**Comment by tricoder42** [comment]
Fixed and released in **2.0.7**. Thanks for reporting!

**Comment by Kernle32DLL** [comment]
Fix confirmed, thanks for the quick response :)
